app/views/repositories/empty_index.html.erb

```diff
@@ -16,6 +16,27 @@
   </div>
 <% end %>
 
+<div class="content-pane flexible <%= params[:archived] ? :archived : :active %> repositories-index"
+      data-readonly="<%= !current_user.is_admin_of_team?(current_team) %>">
+  <div class="content-header">
+    <div class="title-row">
+      <h1 data-view-mode="archived"><i class="fas fa-archive"></i>&nbsp;<%= t('libraries.index.head_title_archived') %></h1>
+      <%= render layout: "shared/view_switch", locals: { disabled: false } do %>
+        <li class="view-switch-archived" data-view-mode="active">
+          <i class="fas fa-archive button-icon"></i>
+          <%= t('libraries.index.switch_view.archived') %>
+        </li>
+        <li class="view-switch-active" data-view-mode="archived">
+          <i class="fas fa-rocket button-icon"></i>
+          <%= t('libraries.index.switch_view.active') %>
+        </li>
+      <% end %>
+    </div>
+  </div>
+  <div class="content-body">
+  </div>
+</div>
```

Comment:
If this was copy-pasted from somewhere, it would be better to make a partial and render the partial at both locations.